lib/spec/router.sw-spec.js

```diff
@@ -206,4 +206,116 @@ describe('Router instances', function () {
     });
   });
 
+
+  describe('Reworked matching algorithm (#1165858)', function() {
+    describe('match()', function() {
+      var mw = function() {};
+
+      it('should match trivial registrations (*)', function() {
```

Comment:
I would not call a path with a wildcard a trivial registration, can you reword this test, please?